### DISCUSSION TITLE
Feature Request: Could you please develop a function to import word files?

### DISCUSSION THREAD

**Discussion by zd178** [ueberdosis/tiptap#5200] [Feature Requests]
### Description

During the demand analysis process, we found that some users needed to import formatted Word files into the editor.

### Use Case

After the user clicks the button, he can select the word file and import it into the editor, and the format can be completely retained.

### Type

New extension

**Comment by nperez0111** [comment]
We have this on our roadmap. But it will likely be a paid pro extension because it is a lot of work to parse (and eventually output) word documents.

This will take some time but hopefully within the year
